pkg/pillar/hypervisor/xen.go

```diff
@@ -493,7 +504,7 @@ func (ctx xenContext) LookupByName(domainName string, domainID int) (int, error)
 
 // Perform xenstore write to disable all of these for all VIFs
 // feature-sg, feature-gso-tcpv4, feature-gso-tcpv6, feature-ipv6-csum-offload
-func (ctx xenContext) Tune(domainName string, domainID int, vifCount int) error {
+func disableVifOffload(domainName string, domainID int, vifCount int) error {
```

Comment:
Doesn't look like this function is used since you use the script to do the work.

Reply:
Wait a sec @eriknordmark -- this is something I don't see in my latest PR -- where are you seeing this?